.github/ISSUE_TEMPLATE/prd.yml

```
@@ -0,0 +1,27 @@
+      placeholder: Bullet list of goals and non‑goals
+  - type: textarea
+    id: ux
+    attributes:
+      label: UX Scope (Figma)
+      placeholder: Link frames and list screens/components touched
+  - type: textarea
+    id: acceptance
+    attributes:
+      label: Acceptance Criteria
+      placeholder: - [ ] Scenario → Expected result
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)